nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+const reservations = require("./../data/reservations.json");
+
+router.get("/", async (req, res) => {
+  try {
+    res.json(reservations);
+  } catch (error) {
+    throw error;
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const result = getReservation(parseInt(req.params.id));
+    Object.keys(result).length === 0
+      ? res.status(404).json({ error: `${req.params.id} not found` })
+      : res.status(200).json(result);
+  } catch (error) {
+    throw error;
+  }
+});
+
+const getReservation = (id) => {
+  return reservations.filter((reservation) => reservation.id === id);
```

Comment:
Same comment as the getMeal function

Reply:
Changed it to  return reservations.find((review) => review.id === id);